<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  
  <param name="yolov5_path"       value="$(find yolo_ros)/yolov5"/>
  <param name="use_cpu"           value="true" />
  <param name="use_half"           value="true" />

  <!-- Start yolov5 and ros wrapper -->
  <node pkg="yolo_ros" type="yolo_v5.py" name="yolo_ros" output="screen" >
    <param name="weight_path"       value="$(find yolo_ros)/weights/v5s_A_B_c_all_right_640_640_no_1280_720_n.onnx"/>
    <param name="color_image_topic"       value="/camera/color/image_raw" />
    <param name="depth_image_topic"       value="/camera/aligned_depth_to_color/image_raw" />
    <param name="calibrate_flag"       value="/coordinate_system_conversion/CalibrateFlag" />
    <param name="pub_topic"         value="/yolov5/BoundingBoxes" />
    <param name="conf"              value="0.1" />
  </node>

</launch>
